app/assets/stylesheets/application.scss

```diff
@@ -7,16 +7,7 @@ $govuk-compatibility-govuktemplate: true;
 @import 'govuk_publishing_components/all_components';
 
 // govuk_frontend_toolkit mixins
-@import "colours";
```

Comment:
I'm a bit confused by this commit. I can find examples of CSS that looks like it's using govuk-frontend-toolkit still, e.g: [the error summary CSS](https://github.com/alphagov/frontend/blob/master/app/assets/stylesheets/helpers/_error-summary.scss#L4) and the old grid classes (`grid-row`).  I think those need to be updated to use GOVUK Frontend

Reply:
Dang it, I knew I forgot to go back and check something. Yes, I spotted that too and was confused by how it was even working. Thanks. I'll go back and remove/convert the remaining stuff.